.github/workflows/maven.yml

```diff
@@ -7,13 +7,9 @@ name: Maven CI
 
 on:
   push:
-    branches:
```

Comment:
Why did that change?

Reply:
With `branches: main`, CI ran only for main branches, so if you make any changes locally it didn't trigger CI in that branch in case anyone would want. :) 
This is not a major concern I think.

Reply:
WDYM by "locally"? I don't think we want to build other branches than "main" (and PRs against main), do we?

Reply:
I meant in the case of fork's branches. For, eg my branch `DBZ-3118`.

Reply:
@ani-sha, sorry for the long radio silence, this one fell a bit through the cracks. So in general, there should be no dev branch like DBZ-3118 in the upstream repo, but this should be in your fork (which it is). So I think this part should be reverted, so that only pushes to upstream/main trigger a build (we could think about doing this for branches like 1.5, 1.4, etc., but the need is low and we should discuss this more holistically).

In the same spirit, the change on lines 15/16 also should be reverted, as only PRs against main should be built.

Reply:
@gunnarmorling Hey, thanks for that clarification! :smiley: 
Reverted back the changes.